meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/stm32mp15-eval-sec/lmp.cfg

```diff
@@ -0,0 +1,26 @@
+CONFIG_BOOTCOMMAND="if test ${boot_device} = serial || test ${boot_device} = usb; then stm32prog ${boot_device} ${boot_instance}; else; setenv verify 1; source ${scriptaddr}; reset; fi;"
```

Comment:
On the -sec target boot command, we don't check for serial boot as there are mfgtools to handle provisioning.

Reply:
right, will fix